lib/rolodex/writers/file_writer.ex

```diff
@@ -1,14 +1,16 @@
 defmodule Rolodex.Writers.FileWriter do
   @behaviour Rolodex.Writer
 
-  alias Rolodex.Config
+  alias Rolodex.{Config, WriterConfig}
 
   @impl Rolodex.Writer
   def init(config) do
-    path = fetch_file_path(config)
-
-    File.touch(path)
-    File.open(path, [:write])
+    with {:ok, file_name} <- fetch_file_name(config),
+         {:ok, cwd} <- File.cwd(),
+         full_path <- cwd <> "/" <> file_name,
```

Comment:
you're gonna hate me but https://hexdocs.pm/elixir/Path.html#join/1 because of `*~ windows ~*`

Reply:
Ahhhhh no good catch thank you!